src/daemon/database/actions.rs

```diff
@@ -713,7 +783,7 @@ pub fn db_update_vault_status(db_path: &Path, db_vault: &DbVault) -> Result<(),
         if all_signed {
             db_tx.execute(
                 "UPDATE vaults \
-                 SET status = (?1), updated_at = strftime('%s','now') \
+                 SET status = (?1), secured_at = ifnull(secured_at, strftime('%s','now')), delegated_at = ifnull(delegated_at, strftime('%s','now')) \
```

Comment:
Good catch to set `ifnull()` for `secured_at`, however `delegated_at` must always be NULL here so let's not have it as it could cover a future bug.